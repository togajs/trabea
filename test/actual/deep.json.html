<!DOCTYPE html>
<html class="no-js" lang="en-us">
<head>
    <title>Documentation</title>
    <style>
        * {
            -webkit-box-sizing: border-box;
            -moz-box-sizing: border-box;
            box-sizing: border-box;
            margin: 0;
            padding: 0;
        }
        html {
            font: 12px/1 sans-serif;
        }
        code {
            font: 0.9rem/1 monospace;
        }
        pre {
            overflow-x: auto;
            white-space: pre;
        }
        td, th {
            text-align: left;
            vertical-align: top;
        }
        .blocks {
            font-size: 0;
            position: relative;
        }
        .blocks:after {
            background: #efd;
            bottom: 0;
            content: ' ';
            display: block;
            left: 50%;
            position: absolute;
            right: 0;
            top: 0;
            z-index: 1;
        }
        .blocks > * {
            display: inline-block;
            font-size: 1rem;
            padding: 1rem;
            margin: 1rem 0;
            position: relative;
            min-height: 1rem;
            vertical-align: top;
            width: 50%;
            z-index: 2;
        }
        .site-hd {
            background: #def;
            padding: 1rem;
        }
    </style>
</head>
<body>
    <div class="site">
        <div class="site-hd">
            
                <h1>Documentation</h1>
            
        </div>

        <div class="site-bd">
            
            
        <div class="blocks">
            <div class="comment">/test/fixtures/deep.json</div>
            
                <div class="code">
    <pre><code class="lang-json"></code></pre>
</div>

            
                <div class="comment">
    <h1 id="title">Title</h1>
<p>Long description that spans multiple
lines and even has other markdown
type things.</p>
<p>Like more paragraphs.</p>
<ul>
<li>Like</li>
<li>Lists</li>
</ul>
<pre><code class="lang-js">    var code = &#39;samples&#39;;
</code></pre>

    <table>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description.</p>
</td>
            </tr>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description that is really long
  and wraps to other lines.</p>
</td>
            </tr>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description that is really long
  and wraps to other lines.</p>
<p>  And has line breaks, etc.</p>
</td>
            </tr>
        
            <tr>
                <th>example</th>
                <td><code></code></td>
                <td><code></code></td>
                <td><pre><code>var foo = &#39;bar&#39;;
</code></pre></td>
            </tr>
        
            <tr>
                <th>tag</th>
                <td><code></code></td>
                <td><code></code></td>
                <td><p>Foo bar.</p>
</td>
            </tr>
        
            <tr>
                <th>tag</th>
                <td><code></code></td>
                <td><code></code></td>
                <td></td>
            </tr>
        
    </table>
</div>

            
                <div class="code">
    <pre><code class="lang-json">

var foo = &#x27;bar&#x27;;

</code></pre>
</div>

            
                <div class="comment">
    <h1 id="title">Title</h1>
<p>Long description that spans multiple
lines and even has other markdown
type things.</p>
<p>Like more paragraphs.</p>
<ul>
<li>Like</li>
<li>Lists</li>
</ul>
<pre><code class="lang-js">    var code = &#39;samples&#39;;
</code></pre>

    <table>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description.</p>
</td>
            </tr>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description that is really long
  and wraps to other lines.</p>
</td>
            </tr>
        
            <tr>
                <th>arg</th>
                <td><code>{Type}</code></td>
                <td><code>name</code></td>
                <td><p>Description that is really long
  and wraps to other lines.</p>
<p>  And has line breaks, etc.</p>
</td>
            </tr>
        
            <tr>
                <th>example</th>
                <td><code></code></td>
                <td><code></code></td>
                <td><pre><code>var foo = &#39;bar&#39;;
</code></pre></td>
            </tr>
        
            <tr>
                <th>tag</th>
                <td><code></code></td>
                <td><code></code></td>
                <td><p>Foo bar.</p>
</td>
            </tr>
        
            <tr>
                <th>tag</th>
                <td><code></code></td>
                <td><code></code></td>
                <td></td>
            </tr>
        
    </table>
</div>

            
                <div class="code">
    <pre><code class="lang-json">
</code></pre>
</div>

            
        </div>
    
        </div>

        <div class="site-ft">
            
            
        </div>
    </div>
</body>
</html>

