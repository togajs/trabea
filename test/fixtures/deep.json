{
    "type": "Document",
    "blocks": [
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "<h1 id=\"title\">Title</h1>\n<p>Long description that spans multiple\nlines and even has other markdown\ntype things.</p>\n<p>Like more paragraphs.</p>\n<ul>\n<li>Like</li>\n<li>Lists</li>\n</ul>\n<pre><code class=\"lang-js\">    var code = &#39;samples&#39;;\n</code></pre>\n",
            "tags": [
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description.</p>\n"
                },
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description that is really long\n  and wraps to other lines.</p>\n"
                },
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description that is really long\n  and wraps to other lines.</p>\n<p>  And has line breaks, etc.</p>\n"
                },
                {
                    "tag": "example",
                    "description": "<pre><code>var foo = &#39;bar&#39;;\n</code></pre>"
                },
                {
                    "tag": "tag",
                    "description": "<p>Foo bar.</p>\n"
                },
                {
                    "tag": "tag"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n\nvar foo = 'bar';\n\n"
        },
        {
            "type": "Comment",
            "description": "<h1 id=\"title\">Title</h1>\n<p>Long description that spans multiple\nlines and even has other markdown\ntype things.</p>\n<p>Like more paragraphs.</p>\n<ul>\n<li>Like</li>\n<li>Lists</li>\n</ul>\n<pre><code class=\"lang-js\">    var code = &#39;samples&#39;;\n</code></pre>\n",
            "tags": [
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description.</p>\n"
                },
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description that is really long\n  and wraps to other lines.</p>\n"
                },
                {
                    "tag": "arg",
                    "type": "{Type}",
                    "name": "name",
                    "description": "<p>Description that is really long\n  and wraps to other lines.</p>\n<p>  And has line breaks, etc.</p>\n"
                },
                {
                    "tag": "example",
                    "description": "<pre><code>var foo = &#39;bar&#39;;\n</code></pre>"
                },
                {
                    "tag": "tag",
                    "description": "<p>Foo bar.</p>\n"
                },
                {
                    "tag": "tag"
                }
            ]
        },
        {
            "type": "Code",
            "contents": "\n"
        }
    ]
}