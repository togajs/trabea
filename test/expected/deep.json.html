<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <title>foo</title>
</head>
<body>
    <div>foo</div>
    <pre class="language-javascript"><code>alert('foo');</code></pre>
</body>
</html>
